modules/ui/success.js

```diff
@@ -32,14 +32,20 @@ export function uiSuccess(context) {
         let ociFeatures = {};
 
         Object.values(ociResources).forEach(resource => {
-          const feature = loco.resolveLocationSet(resource.locationSet).feature;
-          let ociFeature = ociFeatures[feature.id];
-          if (!ociFeature) {
-            ociFeature = JSON.parse(JSON.stringify(feature));  // deep clone
-            ociFeature.properties.resourceIDs = new Set();
-            ociFeatures[feature.id] = ociFeature;
+          let feature;
+          try {
```

Comment:
Also this is wrapped in a `try` now because it can potentially fail to resolve a locationSet with a code from a newer version of country-coder.  (Some of them include things like `gb-nir`, `gb-sct` now)